UnitsNet/UnitDefinitions/Duration.json

```diff
@@ -162,6 +162,18 @@
           "Abbreviations": [ "нс" ]
         }
       ]
+    },
+    {
+      "SingularName": "Cycle",
```

Comment:
Another thing that could be convenient, is to have a way to obtain the `Frequency` or the time period of the utility frequency at a particular country, something like `Frequency usUtilityFrequency = Frequency.GetUtilityFrequencyForCountry("US");`